app/models/orchestration_template.rb

```diff
@@ -70,6 +73,11 @@ def content=(text)
     self.md5 = calc_md5(content)
   end
 
+  # Determines if validation for md5 uniqueness is done
+  def unique_md5?
+    !draft?
```

Comment:
If it has an `ems_ref` or `ems_id`, we don't need to restrict the uniquness
`!draft? && !ems_ref`